frontend-implicaction/src/app/core/services/api-endpoints.service.ts

```diff
@@ -226,6 +227,21 @@ export class ApiEndpointsService {
   /**
    * Companies
    */
+  getAllCompanyByCriteriaEndpoint(pageable: Pageable, criteria: CompanyCriteriaFilter): string {
+    // on merge les filtres et les attributs de pagination
+    const objectParam = {
+      ...criteria,
+      rows: pageable.rows,
+      page: pageable.page,
+      sortBy: pageable.sortBy,
+      sortOrder: pageable.sortOrder
+    };
+    return ApiEndpointsService.createUrlWithQueryParameters(
+      Uris.COMPANIES.BASE_URI,
+      (qs: QueryStringParameters) => {
+        this.buildQueryStringFromFilters(objectParam, qs);
+      });
```

Comment:
```suggestion
      (qs: QueryStringParameters) => this.buildQueryStringFromFilters(objectParam, qs)
  );
```

Si je dis pas de bétise ça doit fonctionner